hack/release.yaml

```diff
@@ -73,3 +73,7 @@ ovsCni:
   image: ovs-cni-plugin
   repository: nvcr.io/nvstaging/mellanox
   version: 61fd27c
+rdmaCni:
+  image: rdma-cni
+  repository: ghcr.io/k8snetworkplumbingwg
+  version: latest
```

Comment:
I think we should have a specific version here, no?

Reply:
I would like to keep `latest` for beta releases to simplify testing as it contains multi-arch images. If you think it's better to use some tag for beta release I'll update PR

Reply:
Ack.
Just let's make sure that we do not release with `latest`